### PR TITLE
fix(Physics Wallah): Website updated

### PR DESCRIPTION
**Pull request by armup31**
## Description 
The website had changed a few things so the thing wasn't working, the code is mildly updated and everything works

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

</details>

### CODE REVIEW

File: websites/P/Physics Wallah/presence.ts [hunk 1]
```diff
@@ -64,21 +66,25 @@ presence.on("UpdateData", async () => {
...
-				presenceData.details = urlParams.get("subject");
-				presenceData.state = urlParams.get("topic");
+		} else if (fullurl.includes("subject-topics")) {
+			if (fullurl.includes("chapterId")) {
+				presenceData.details = JSON.parse(
+					localStorage.getItem("SCHEDULE_DETAILS")
+				).tags[0];
+				presenceData.state = JSON.parse(
+					localStorage.getItem("SCHEDULE_DETAILS")
+				).subject.name;
```
Comment (Bas950) [inline websites/P/Physics Wallah/presence.ts:76]: json parse it once, save as a variable, then use the .tags / .subject

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/P/Physics Wallah/presence.ts [hunk 2]
```diff
@@ -15,7 +15,8 @@ presence.on("UpdateData", async () => {
 			startTimestamp: browsingTimestamp,
 			type: ActivityType.Watching,
 		},
-		{ pathname, href, search } = document.location,
+		{ pathname } = document.location,
+		fullurl = document.URL,
```
Comment (theusaf) [inline websites/P/Physics Wallah/presence.ts:19]: Use `document.location.href` instead of `document.URL` for code consistency

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by armup31** [comment]
don't merge, something broke again the website is getting kinda buggy

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/P/Physics Wallah/presence.ts [hunk 3]
```diff
@@ -15,8 +15,10 @@ presence.on("UpdateData", async () => {
 			startTimestamp: browsingTimestamp,
 			type: ActivityType.Watching,
 		},
-		{ pathname, href, search } = document.location,
-		privacyMode = await presence.getSetting<boolean>("privacy");
+		{ pathname, href } = document.location,
+		fullurl = href,
```
[the remark this answers is not in the document]
Comment (armup31) [reply websites/P/Physics Wallah/presence.ts:19]: probably i was testing out something and i forgot to remove it, can't we just ignore it?

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**